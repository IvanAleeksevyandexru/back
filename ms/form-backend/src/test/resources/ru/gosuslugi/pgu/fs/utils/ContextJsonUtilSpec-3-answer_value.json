{
  "region": "Москва",
  "city": "",
  "district": "",
  "town": "",
  "inCityDist": "",
  "street": "Варшавское",
  "additionalArea": "",
  "additionalStreet": "",
  "house": "141А",
  "houseCheckbox": false,
  "houseCheckboxClosed": false,
  "building1": "3",
  "building2": "",
  "apartment": "999",
  "apartmentCheckbox": false,
  "apartmentCheckboxClosed": false,
  "index": "117405",
  "geoLat": "55.584645",
  "geoLon": "37.603487",
  "fullAddress": "117405, г. Москва, ш. Варшавское, д. 141А, к. 3, кв. 999",
  "addressStr": "г. Москва, ш. Варшавское",
  "lat": "55.584645",
  "lng": "37.603487",
  "fiasCode": "4ff7be6b-e016-49b1-aee7-cd98c1f15e2a",
  "okato": "45296597000",
  "hasErrors": 0,
  "kladrCode": "7700000000004761533",
  "apartmentKladr":"7700000000004761533",
  "streetKladr": "7700000000004761533",
  "regionKladr": "77000000000004",
  "regionCode": "77"
}