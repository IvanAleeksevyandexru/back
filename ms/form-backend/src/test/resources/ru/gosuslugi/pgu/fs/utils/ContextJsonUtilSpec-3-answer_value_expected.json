{
  "additionalArea": "",
  "additionalAreaType": "",
  "additionalAreaShortType": "",
  "additionalStreet": "",
  "additionalStreetType": "",
  "additionalStreetShortType": "",
  "addressStr": "г. Москва, ш. Варшавское",
  "apartment": "999",
  "apartmentCheckbox": false,
  "apartmentCheckboxClosed": false,
  "apartmentShortType": "кв",
  "apartmentType": "квартира",
  "building1": "3",
  "building1ShortType": "к",
  "building1Type": "корпус",
  "building2": "",
  "building2Type": "",
  "building2ShortType": "",
  "city": "",
  "cityType": "",
  "cityShortType": "",
  "district": "",
  "districtType": "",
  "districtShortType": "",
  "fiasCode": "4ff7be6b-e016-49b1-aee7-cd98c1f15e2a",
  "apartmentFias":"",
  "regionFias": "",
  "streetFias": "",
  "houseFias": "",
  "townFias": "",
  "fullAddress": "117405, г. Москва, ш. Варшавское, д. 141А, к. 3, кв. 999",
  "geoLat": "55.584645",
  "geoLon": "37.603487",
  "hasErrors": 0,
  "house": "141А",
  "houseCheckbox": false,
  "houseCheckboxClosed": false,
  "houseShortType": "д",
  "houseType": "дом",
  "inCityDist": "",
  "inCityDistType": "",
  "inCityDistShortType": "",
  "index": "117405",
  "kladrCode": "7700000000004761533",
  "apartmentKladr":"7700000000004761533",
  "streetKladr": "7700000000004761533",
  "regionKladr": "77000000000004",
  "townKladr": "",
  "houseKladr": "",
  "lat": "55.584645",
  "lng": "37.603487",
  "okato": "45296597000",
  "region": "Москва",
  "regionCode": "77",
  "regionShortType": "г",
  "regionType": "город",
  "street": "Варшавское",
  "streetShortType": "ул",
  "streetType": "улица",
  "town": "",
  "townType": "",
  "townShortType": "",
  "oktmo":"",
  "postalCode":"",
  "oktmoName":""
}